plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1890,6 +1895,46 @@ private static void checkNoMetadataUpdates(DeltaLakeCommitSummary commitSummary)
         }
     }
 
+    private static void checkIfCommittedAddedFilesConflictWithCurrentOperation(TupleDomain<DeltaLakeColumnHandle> enforcedPartitionConstraint, IsolationLevel isolationLevel, DeltaLakeCommitSummary commitSummary)
+    {
+        List<AddFileEntry> addedFiles = commitSummary.getIsBlindAppend().orElse(false)
+                ? List.of() // Do not conflict with blind appends
```

Comment:
Explain the reasoning why .

Thank you @pajaks  for pointing out why do we actually conflict with non-blind inserts for WriteSerializable.